Test/civl/inductive-sequentialization/NBuyer.bpl

```diff
@@ -255,7 +206,8 @@ returns ({:pending_async "SellerFinish","FirstBuyer","MiddleBuyer","LastBuyer"}
 modifies QuoteCH;
 {
   assert Init(pids, ReqCH, QuoteCH, RemCH, DecCH, contribution);
-  QuoteCH := (lambda i:int :: (lambda q:int :: if buyerID(i) && q == price then 1 else 0));
+  assume {:add_to_pool "contribution", contribution} true;
```

Comment:
Would it make sense to attach the `add_to_pool` attribute to the assertion or better keep it separate? Wondering if there is any convention.

Reply:
I have recently realized that putting ```:add_to_pool``` annotations on asserts in the body of atomic actions is not being handled correctly since I added the ```HoistAsserts``` code. This code is not propagating the attributes on the asserts as it hoists them. 

I am unsure what to do. I am thinking about restricting the scope of ```add_to_pool``` even further in the documentation allowing them to be used only on assume commands.